MapboxAndroidSDK/src/main/java/com/mapbox/mapboxsdk/views/MapView.java

```diff
@@ -1792,16 +1792,10 @@ private UserLocationOverlay getOrCreateLocationOverlay() {
     }
 
     /**
-     * Show or hide the user location overlay
+     * Show the user location overlay
      */
     public MapView setUserLocationEnabled(final boolean value) {
```

Comment:
I see why this was done, but I think it's important to be able to toggle user location when _not_ in proximity mode. I'm just going to merge this in for now and make an issue.